defaults/main.yml

```diff
@@ -527,6 +527,12 @@ gitlab_runner__cache_secret_key: ''
 gitlab_runner__cache_bucket_name: ''
 
                                                                    # ]]]
+# .. envvar:: gitlab_runner__cache_bucket_location [[[
+#
+# Cache bucket location
+gitlab_runner__cache_bucket_location: 'us-east-1'
```

Comment:
Actually not sure if it'll interfere. I'll just leave it blank since the rest of the cache variables are blank and you'll have to fill them out anyways.